software/webapp/src/test/java/org/apache/brooklyn/entity/dns/geoscaling/GeoscalingWebClientTest.java

```diff
@@ -100,12 +106,9 @@
 ... 31 more
  */
 @Test(groups="Broken", enabled=false)
```

Comment:
Yes, for reasons described here: https://issues.apache.org/jira/browse/BROOKLYN-200